code/game/atom/_atom.dm

```diff
@@ -229,6 +229,8 @@
 		return TRUE
 	if(mover.throwing && (pass_flags_self & LETPASSTHROW))
 		return TRUE
+	if(SEND_SIGNAL(src, COMSIG_ATOM_CAN_ALLOW_THROUGH, mover, border_dir) & COMSIG_FORCE_ALLOW_THROUGH)
+		return TRUE
```

Comment:
Is this signal needed, doesn't COMSIG_ATOM_TRIED_PASS achieve the same?

Reply:
I'll be honest I took this from our wallening code and forgot to check if anyone else had added a duplicate signal in the meanwhile (it was me (I added it))